clang/include/clang/CConv/CheckedRegions.h

```diff
@@ -66,6 +66,7 @@ class CheckedRegionFinder : public RecursiveASTVisitor<CheckedRegionFinder>
     bool VisitVarDecl(VarDecl *VD);
     bool VisitParmVarDecl(ParmVarDecl *PVD);
     bool VisitMemberExpr(MemberExpr *E);
+    bool VisitDeclRefExpr(DeclRefExpr*);
```

Comment:
When was this needed/coming up? I understand from https://clang.llvm.org/doxygen/classclang_1_1DeclRefExpr.html#details that this a "reference to a declaration in an expression". Perhaps we were missing stuff in `for` loops? What else?

Reply:
Deals with variables in expressions.